src/styles/blocks/calendar.scss

```
@@ -0,0 +1,72 @@
+$cell-width: 100px;
+$cell-height: $cell-width;
+$month-days: 31;
+$cell-gap: 1px;
+$transform-shift: -20px;
+$week-day:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+.calendar {
+  box-sizing: border-box;
+  width: $cell-width * 7 + $cell-gap * 6;
+  max-height: 100vh;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  margin: 0 auto;
+  align-items: center;
+  vertical-align: middle;
+  align-self: center;
+
+  &__cell {
+    box-sizing: border-box;
+    width: $cell-width;
+    height: $cell-height;
+    padding: 10px;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    background-color: #eee;
+    border: 1px solid black;
+
+    transition: transform 0.5s, background-color 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+
+      transform: translateY($transform-shift);
+      transition: transform 0.5s, background-color 0.5s;
```

No need to repeat the same code above
```suggestion
      background-color: pink;
```